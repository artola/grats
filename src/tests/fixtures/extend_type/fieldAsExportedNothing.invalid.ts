/** @gqlType */
class SomeType {
  // No fields
}

/** @gqlField */
export const greeting;
