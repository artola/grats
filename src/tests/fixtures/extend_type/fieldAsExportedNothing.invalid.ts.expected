-----------------
INPUT
----------------- 
/** @gqlType */
class SomeType {
  // No fields
}

/** @gqlField */
export const greeting;

-----------------
OUTPUT
-----------------
src/tests/fixtures/extend_type/fieldAsExportedNothing.invalid.ts:7:1 - error: Expected `@gqlField` on variable declaration to be attached to an arrow function.

7 export const greeting;
  ~~~~~~~~~~~~~~~~~~~~~~
